docs/resources/samples/performance.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Performance optimization'
```

Comment:
Let's center the article's title and description around the sample. There's already a perf article in the doc set.